tests/Feature/SecurityTest.php

```
@@ -11,315 +9,293 @@
-    private function jobServerId(UpdateJob $job): int
-    {
-        $reflection = new \ReflectionClass($job);
-        $property = $reflection->getProperty('server');
-        $property->setAccessible(true);
-
-        /** @var Server $server */
-        $server = $property->getValue($job);
-
-        return $server->id;
-    }
+    $reflection = new ReflectionClass($job);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '293', column '23'). (undefined)

(MissingImport)